runtime/oti/j9vm.hdf

```diff
@@ -1351,4 +1351,18 @@ typedef UDATA (* lookupNativeAddressCallback)(struct J9VMThread *currentThread,
 		<struct>J9VirtualThreadEndEvent</struct>
 		<data type="struct J9VMThread*" name="currentThread" description="current thread" />
 	</event>
+
+	<event>
+		<name>J9HOOK_VIRTUAL_THREAD_MOUNT</name>
```

Comment:
Most (if not all) other hooks have `_VM_` in their name.

Reply:
@dipak-bagadiya We should add `_VM_` to the hook events to match the existing formatting.

Reply:
added _VM_ to the hook events to match the existing formatting.